client/src/details/connection/HttpHeader.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include <correlation_vector/correlation_vector.h>
 
+constexpr const char* c_userAgent = "Microsoft-SFSClient/" SFS_VERSION;

```

Comment:
it's not uncommon for a user agent to also send up architecture info as well -- I think that might be useful here too, check with service folks.

e.g.

`Microsoft-SFSClient/1.0 (Windows 10.0; x64)`

Reply:
Nice. I've added that with the "best effort" APIs we have for the expected OSes. Updated description with more.

Examples:
- Microsoft-SFSClient/1.0.0 (Windows NT 10.0; x64)
- Microsoft-SFSClient/1.0.0 (Ubuntu 22.04.3 LTS; x86_64)